sdk/cosmos/azure-cosmos-spark_3-2_2-12/README.md

```diff
@@ -47,6 +47,11 @@ https://github.com/Azure/azure-sdk-for-java/issues/new
 | 4.6.1     | 3.2.0 - 3.2.1            | 8                    | 2.12                     | 10.\*                         |
 | 4.6.0     | 3.2.0 - 3.2.1            | 8                    | 2.12                     | 10.\*                         |
 
+#### azure-cosmos-spark_3-3_2-12
+| Connector     | Supported Spark Versions | Minimum Java Version | Supported Scala Versions | Supported Databricks Runtimes |
+|---------------|--------------------------| -------------------- | -----------------------  |-------------------------------|
+| 4.15.0-beta.1 | 3.3.0                    | 8                    | 2.12                     | 11.\*                         |
```

Comment:
Are we planning to publish this beta to Maven? If not this should only be added when we actually publish a Spark 3.3 supporting version to Maven.

Reply:
@kushagraThapar your thoughts on this? I assume we will **not** publish to maven, so we should remove entry here until we do?

Reply:
I think we closed on publishing a beta to Maven - so your versions make sense.